README.md

```diff
@@ -83,6 +87,16 @@ website is unknown.
 The `facilities` member may contain `parking` or `accessibility` members. Each of these members holds an array of objects with properties `name` and `exists`.
 The `facilities` member is optional, as are the child members. When a child member is present it will always contain at least one item, and the `name` and `exists` members are always populated.
 
+### Services
+
+The `services` member is optional and may contain the members `epsEnabled`, `moreInformaton`, or `entries`. 
+`epsEnabled` is optional and will be `true` if the practice provides the Electronic Prescription Service.
+`moreInformation` is optional free text.
+
+The `entries` array is always present and contains objects with members `title`, `code`, `availabilityTimes`, `introduction`, `gpReferralRequired` and `deliverer`. 
+`gpReferralRequired` will be either `true` or `false`. 
+`availabilityTimes` and `introduction` are optional.
```

Comment:
The contents of the members are mostly self-explanatory but what would be in `availabilityTimes`?
Is `code` anything special or just some arbitrary code from the source data?